model/client4.go

```diff
@@ -7755,6 +7755,19 @@ func (c *Client4) GetProductLimits() (*ProductLimits, *Response, error) {
 	return productLimits, BuildResponse(r), nil
 }
 
+func (c *Client4) GetCloudUsageForMessages() (*MessagesUsage, *Response, error) {
+	r, err := c.DoAPIGet(c.cloudRoute()+"/usage/messages", "")
+	if err != nil {
+		return nil, BuildResponse(r), err
+	}
+	defer closeBody(r)
+
+	var usage *MessagesUsage
+	json.NewDecoder(r.Body).Decode(&usage)
```

Comment:
We should check the error returned. Most of the other client code doesn't check marshalling/unmarshalling errors, which is something we need to fix eventually.